app/components/list-widget/component.js

```diff
@@ -162,26 +55,28 @@ export default Ember.Component.extend({
                 number: raw_datum.doc_count,
                 name: raw_datum.key
             };
-        }).filter((datum) => {
-            if (this.get("chartType") === "tagsList") {
-                if (tag_blacklist.indexOf(datum.name) >= 0) {
-                    //return false;
-                }
-            }
-            return true;
         }));
     },
+
+    fetchAgentDetails: async function(data) {
```

Comment:
This could use a refactor at some point to make this fn reusable and clean up the init fn.